web-client/public/js/graph.js

```diff
@@ -859,8 +834,8 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
     }
 
     var dblclick = function (d) {
-        d3.event.stopPropagation();
-        d3.select(this).classed("fixed", d.fixed = false);
+        d.fx = null;
+        d.fx = null;
```

Comment:
Looks like a duplicate line.

Reply:
yes good catch @dondi it should have been `d.fx` and `d.fy`